README.md

```diff
@@ -20,56 +20,33 @@ Prerequisites
 -------------
 
 
-### Linux
+### Linux and Mac OS 
 
 For Ginkgo core library:
 
-*   _cmake 3.1+_
-*   C++11 compliant compiler, one of:
-    *   _gcc 5.4.0+_
-    *   _clang 3.3+_ (__TODO__: verify, works with 5.0)
-
-The Ginkgo CUDA module has the following __additional__ requirements:
-
-*   _cmake 3.10+_
-*   _CUDA 7.0+_ (__TODO__: verify, works with 8.0)
-*   Any host compiler restrictions your version of CUDA may impose also apply
-    here. For the newest CUDA version, this information can be found in the
-    [CUDA installation guide](http://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#system-requirements)
-
-In addition, if you want to contribute code to Ginkgo, you will also need the
-following:
-
-*   _clang-format 5.0.1+_ (ships as part of _clang_)
-
-
-### Mac OS
-
-For Ginkgo core library:
-
-*   _cmake 3.1+_
+*   _cmake 3.8+_
 *   C++11 compliant compiler, one of:
-    *   _gcc 5.4.0+_ (__TODO__: verify)
-    *   _clang 3.3+_ (__TODO__: verify)
+    *   _gcc 5.5.0+_
```

Comment:
The question with gcc is what does `+` mean? They have a very strange [release history](https://www.gnu.org/software/gcc/releases.html), where a higher number does not necessarily mean the version is newer or has better standard conformance.

Does this mean we only support 7.3, 8.1, 8.2, 6.5, 7.4 and nothing else? I think I compiled at some point with 7.2, which is older. On the other hand, it might be the case that it does not work with 6.1.

We should probably have a separate minimum version requirement for each major version. (E.g works with 5.5+, 7.2+, 6.3+, 8.1+ - these numbers are made up, didn't actually test)

Reply:
What we actually test are more or less the latest ones of each, so getting a full list of these is fairly hard (and making sure that this stays the case). Currently with the CI docker images we test: 5.5, 6.5, 7.3, 8.1. 

On the FineCI server, I tried with 5.3.1 and 6.3.1 and it worked properly. I can add these to the list and if some user finds in the future that due new additions and to some compiler bug it is not true anymore we consider this a bug and fix it. Or update the list after deliberation.